src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -285,8 +285,7 @@ private static bool PlatformAnalysisAllowed(AnalyzerOptions options, Compilation
             string tfmVersion = options.GetMSBuildPropertyValue(MSBuildPropertyOptionNames.TargetFrameworkVersion, compilation) ?? "";
 
             if (tfmIdentifier.Equals(NetCoreAppIdentifier, StringComparison.OrdinalIgnoreCase) &&
-                tfmVersion.StartsWith("v", StringComparison.OrdinalIgnoreCase) &&
-                Version.TryParse(tfmVersion[1..], out var version) &&
+                Version.TryParse(tfmVersion.TrimStart(['v', 'V']), out var version) &&
```

Comment:
An interesting "quirk" of this is that technically `vvvvvvvv5.0` is also valid; but I don't think that's worth adding an explicit test around 😄